pages/login/index.js

```diff
@@ -1,29 +1,119 @@
 import styled from "styled-components";
-import Link from "next/link";
+import {signIn} from "next-auth/react";
+import github from "../../public/images/github.svg";
+import github_logo from "../../public/images/github_logo.svg";
+import Image from "next/image";
+import FeatureBackground from "../../components/styled/FeatureBackground";
+import {useState} from "react";
+import StyledBackground from "../../components/styled/StyledBackground";
 
 export default function Login() {
+  const [email, setEmail] = useState("");
+  const [pass, setPass] = useState("");
+
+  const handleSubmit = e => {
+    e.preventDefault();
+    const form = e.target;
+    const {email, password} = form.elements;
+    setEmail(email.value);
+    setPass(password.value);
+  };
+
+  function magic(event) {
+    const {currentTarget: el, clientX: x, clientY: y} = event;
+    const {top: t, left: l, width: w, height: h} = el.getBoundingClientRect();
+    el.style.setProperty("--posX", x - l - w / 2);
+    el.style.setProperty("--posY", y - t - h / 2);
+  }
+
   return (
     <>
-      <StyledSection>
-        <Link href="/profile">
-          <StyledButton>LOGIN </StyledButton>
-        </Link>
-      </StyledSection>
+      <FeatureBackground onPointerMove={magic} login>
+        <StyledBackground>
+          <StyledForm onSubmit={handleSubmit}>
+            <label for="email" />
+            <StyledInput
+              value={email}
+              placeholder="test@gmail.com"
+              type="email"
+              id="email"
+              name="email"
+            />
+            <label for="password" />
+            <StyledInput
+              value={pass}
+              type="password"
+              placeholder="********"
+              id="password"
+              name="password"
+            />
+            <StyledLoginButton type="submit">LOGIN </StyledLoginButton>
+          </StyledForm>
+          <StyledRegister>
+            Dont have an account? Register{" "}
+            <StyledRegisterButton>here</StyledRegisterButton>!
+          </StyledRegister>
+
+          <StyledButtonGitHub
+            onClick={() =>
+              signIn(undefined, {
+                callbackUrl: "http://localhost:3000/home",
+              })
+            }
+          >
+            <Image alt="github_logo" src={github_logo}></Image>
+            <p>Login with </p>
+            <Image alt="github" src={github}></Image>
+          </StyledButtonGitHub>
+        </StyledBackground>
+        {/*    <Link href="/profile">
+          <StyledButtonMain test>Go to app (without login)</StyledButtonMain>
+        </Link> */}
+      </FeatureBackground>
     </>
```

Comment:
You don't need a fragment for this case. So delete `<>` and `</>`.

Reply:
done!